README.md

```diff
@@ -72,3 +72,20 @@ Swagger UI will be available @ http://127.0.0.1:8000/docs
   }]
 }
 ```
+
+### API request prototype for adding a post
+```json
+{
+  "data": {
+    "type": "posts",
```

Comment:
Imo this type field in excessive, because we know posts will be there, as we send that JSON to /posts/ endpoint. Check how I changed 'GET' prototype. https://github.com/kozlowskimaciej/microblog/pull/20/files